Tests/src/PublicAPITests/SpaceSystemTests.cpp

```diff
@@ -1217,8 +1217,17 @@ CSP_PUBLIC_TEST(CSPEngine, SpaceSystemTests, UpdateUserRolesTest)
 	::UserRoleInfo UpdatedDefaultUserRole	 = {DefaultUserId, SpaceUserRole::Moderator};
 	::UserRoleInfo UpdatedSecondTestUserRole = {AltUserId, SpaceUserRole::Owner};
 
-	UpdateUserRole(SpaceSystem, Space.Id, UpdatedDefaultUserRole);
-	UpdateUserRole(SpaceSystem, Space.Id, UpdatedSecondTestUserRole); // NOTE: This only works because our default test account has superuser access
+	// User Roles should not be changed after update as a owner cannot be modified
+	// This also means a owner cannot be turned into a moderator
+	auto [DefaultResult] = AWAIT_PRE(SpaceSystem, UpdateUserRole, RequestPredicate, Space.Id, UpdatedDefaultUserRole);
+
+	// Update first account role should fail
```

Comment:
I presume that should say 'Update first account role should succeed'?

Reply:
This is a bug in chs as it should only succeed if the role is updated, but as this unblocks PRs seems